sql/update_bene_monthly_proc.sql

```diff
@@ -1,41 +1,63 @@
 CREATE OR REPLACE PROCEDURE update_bene_monthly(
```

Comment:
The CCW data likely includes a lot of rows where all of the data fields are `NULL`; they likely didn't filter those out. We need to: the load script will have to check for and skip year-months where all of the fields are `NULL`.

Reply:
> Do I need to check for null in each month's (13) columns?

Yes. And we also need to check for _`NULL`-like_ values, e.g. if the BFD Pipeline treats "0" in a particular column as a null, then this script should, too.

> If not, what subset of columns makes sense (i.e, just check PTC_CNTRCT_ID and PTD_CNTRCT_ID)?

To frame it a different way: check for nulls in every column except for the bene ID and the date.